pyproject.toml

```diff
@@ -96,7 +96,6 @@ force-exclude = ".venv|/_.*_dict.py$"  # to not do files pre-commit asks for
 [tool.ruff]
 src = ["src"]
 line-length = 214
-target-version = "py310"
```

Comment:
Why remove this?

Reply:
It's in the commit message:
> You have both Ruff's `target-version` and `project.requires-python`. You only need the latter.

Reply:
From https://docs.astral.sh/ruff/settings/#target-version:
> If you're already using a `pyproject.toml` file, we recommend `project.requires-python` instead, as it's based on Python packaging standards, and will be respected by other tools. For example, Ruff treats the following as identical to `target-version = "py38"`:
> ```toml
> [project]
> requires-python = ">=3.8"
> ```

Reply:
Ok, thanks!